tests/mocks/ingest-data/files/mock-analysis-output-file.json

```diff
@@ -27,7 +30,12 @@
   ],
   "fileName": "be53a909-2f0a-4b5c-ba67-c07f390f2f67_qc.bam",
   "cloudUrl": "s3://org-humancellatlas-upload-integration/a610450d-fcc7-49d3-a7f4-b6ed4022059b/be53a909-2f0a-4b5c-ba67-c07f390f2f67_qc.bam",
-  "checksums": null,
+  "checksums": {
+    "sha256": "sha256",
+    "crc32c": "crc32c",
+    "sha1": "sha1",
+    "s3_etag": "s3_etag"
+  },
```

Comment:
These are relatively newer fields that were added in the file metadata document. The test data was probably copied from an old analysis file metadata.